druid-shell/src/backend/windows/window.rs

```diff
@@ -1342,9 +1343,11 @@ impl WindowBuilder {
 
     pub fn set_level(&mut self, level: WindowLevel) {
         match level {
-            WindowLevel::AppWindow | WindowLevel::Tooltip(_) => self.level = Some(level),
+            WindowLevel::AppWindow | WindowLevel::Tooltip(_) | WindowLevel::DropDown(_) => {
```

Comment:
Can you add `Modal` here also?

Reply:
Added `Modal` and removed the `match` statement.